src/roulette.py

```diff
@@ -126,3 +126,124 @@ def get(self, bin: int) -> Bin:
+            wheel.addOutcome(indexes_for_street_bets[1], street_bet)
+            wheel.addOutcome(indexes_for_street_bets[2], street_bet)
+
+    @staticmethod
+    def create_corner_bets(wheel: Wheel) -> None:
+        odds_for_corner_bet = 8
+
+        first_column_numbers = set()
+        for i in range(11):
+            first_column_numbers.add(i * 3 + 1)
+        for i in first_column_numbers:
+            first_column_corner_bet = Outcome(str(i) + "-" + str(i + 1) + "=" + str(i + 3) +
+                                              str(i + 4), odds_for_corner_bet)
+            indexes_for_first_column_corner_bets = i, i + 1, i + 3, i + 4
+            wheel.addOutcome(indexes_for_first_column_corner_bets[0], first_column_corner_bet)
+            wheel.addOutcome(indexes_for_first_column_corner_bets[1], first_column_corner_bet)
+            wheel.addOutcome(indexes_for_first_column_corner_bets[2], first_column_corner_bet)
+            wheel.addOutcome(indexes_for_first_column_corner_bets[3], first_column_corner_bet)
+
+        second_column_number = set()
+        for i in range(11):
+            second_column_number.add(i * 3 + 2)
+        for i in second_column_number:
+            second_column_corner_bet = Outcome(str(i) + "-" + str(i + 1) + "=" + str(i + 3) +
+                                               str(i + 4), odds_for_corner_bet)
+            indexes_for_first_column_corner_bets = i, i + 1, i + 3, i + 4
+            wheel.addOutcome(indexes_for_first_column_corner_bets[0], second_column_corner_bet)
+            wheel.addOutcome(indexes_for_first_column_corner_bets[1], second_column_corner_bet)
+            wheel.addOutcome(indexes_for_first_column_corner_bets[2], second_column_corner_bet)
+            wheel.addOutcome(indexes_for_first_column_corner_bets[3], second_column_corner_bet)
+
+    @staticmethod
+    def create_line_bets(wheel: Wheel) -> None:
+        odds_for_line_bets = 5
+        numbers_to_generate_line_bets = set()
+        for i in range(11):
+            numbers_to_generate_line_bets.add(i * 3 + 1)
+
+        for i in numbers_to_generate_line_bets:
+            line_bet = Outcome(str(i) + "-" + str(i + 1) + "-" + str(i + 2) + "-" + str(i + 3) +
+                               "-" + str(i + 4) + "-" + str(i + 5), odds_for_line_bets)
+            indexes_for_line_bets = (i, i + 1, i + 2, i + 3, i + 4, i + 5,)
+            wheel.addOutcome(indexes_for_line_bets[0], line_bet)
+            wheel.addOutcome(indexes_for_line_bets[1], line_bet)
+            wheel.addOutcome(indexes_for_line_bets[2], line_bet)
+            wheel.addOutcome(indexes_for_line_bets[3], line_bet)
+            wheel.addOutcome(indexes_for_line_bets[4], line_bet)
+            wheel.addOutcome(indexes_for_line_bets[5], line_bet)
+
+    @staticmethod
+    def generate_five_bets(wheel: Wheel) -> None:
+        odds_for_five_bet = 6
+        five_bet = Outcome("00-0-1-2-3", odds_for_five_bet)
```

Comment:
better create `five_bet_name` variable to avoid magic string

Reply:
Done. Thanks.